app/jobs/result_monitor_loader.rb

```diff
@@ -0,0 +1,22 @@
+# Load a result from S3 into the db
+require 'logger'
+class ResultMonitorLoader
+  @queue = :q03_pipeline_run
+  @logger = Logger.new(STDOUT)
+
+  def self.perform(pipeline_run_id, output)
+    @logger.info("Loading #{output} for pipeline run #{pipeline_run_id}")
+    pr = PipelineRun.find(pipeline_run_id)
+    return unless pr.result_status_for(output) == PipelineRun::STATUS_LOADING_QUEUED
```

Comment:
Given that this shouldn't happen, I'd probably print a warning when it does.   Not worth making a change if this is the only change, just a thought for future.

Reply:
Do you also want to exit early if the pipeline run has failed?   Or perhaps not, because you want a failure in postprocessing not to prevent alignment results from getting loaded?

Reply:
ok, changed